package.json

```
@@ -4,11 +4,12 @@
   "description": "Needle is a discord bot that helps you manage your discord threads.",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "rd /s /q dist & tsc",
-    "start": "npm run build && node ./dist/index.js",
-    "dev": "npm run build && node ./scripts/deploy-commands.js && node ./dist/index.js",
-    "undeploy": "npm run build && node ./scripts/deploy-commands.js  --undeploy",
-    "deploy": "npm run undeploy && node ./scripts/deploy-commands.js --global"
+    "build": "tsup",
+    "start": "npm run build && node --enable-source-maps .",
+    "dev": "npm run deploy:dev && tsup --watch --onSuccess \"node --enable-source-maps .",
```

This string isn't terminated, resulting in an error when running `npm run dev`.
```suggestion
    "dev": "npm run deploy:dev && tsup --watch --onSuccess \"node --enable-source-maps .\"",
```

---

🤦